src/views/Profile/index.js

```diff
@@ -420,7 +258,7 @@ export default class Profile extends Component {
                   </Grid>
                   <br />
                   <h1>Please Re-establish Connection</h1>
-                  <h4>People Search has been deactivated due to loss of network.</h4>
+                  <h4>Viewing a public profile has been deactivated due to loss of network.</h4>
```

Comment:
Fixes the name of the current page that the user is on. For some reason, it originally says "People Search" even though the page is the Public Profile